satpy/readers/nc_slstr.py

```diff
@@ -91,15 +94,16 @@ def __init__(self, filename, filename_info, filetype_info):
                                           'rows': CHUNK_SIZE})
         self.nc = self.nc.rename({'columns': 'x', 'rows': 'y'})
         self.channel = filename_info['dataset_name']
-        self.view = 'n'  # n for nadir, o for oblique
+        self.stripe = filename[-5]
+        self.view = filename[-4]
```

Comment:
could we use the filename_info or filetype_info to get this information ? I'm afraid a filename change on the ESA side would break this.

Reply:
I tried your suggestion but failed to get it to work. I asked a bit around and it seems like the file name convention is cast in stone. @caropoulsen might want to comment.

Reply:
Just to be clear this should be possible if the patterns were updated to match the `SL` part of the filename to `{view:1s}{stripe:1s}` then you should be able to get them with `self.filename_info['view']` and `self.filename_info['stripe']`. I am ok leaving this if this is the only suggested change remaining in this PR.